3p/recaptcha.js

```diff
@@ -111,7 +124,7 @@ function initializeIframeMessagingClient(window, grecaptcha, dataObject) {
   iframeMessagingClient.setSentinel(dataObject.sentinel);
   iframeMessagingClient.registerCallback(
       'amp-recaptcha-action',
-      actionTypeHandler.bind(this, grecaptcha)
+      actionTypeHandler.bind(this, window, grecaptcha)
```

Comment:
can we have a proper unit test?

for example, we could fake postMessages to the window object, and stub out the `grecaptcha` and check if the `execute` are correctly invoked.